### PR TITLE
[i18n-<zh>] Translating trainer.md to chinese

### LINKED ISSUE #29726: [i18n-<zh>] Translating trainer.md to chinese
Hi!

Let's bring the documentation to all the <languageName>-speaking community 🌐 (currently 0 out of 267 complete)

Who would want to translate? Please follow the 🤗 [TRANSLATING guide](https://github.com/huggingface/transformers/blob/main/docs/TRANSLATING.md). Here is a list of the files ready for translation. Let us know in this issue if you'd like to translate any, and we'll add your name to the list.

Some notes:

* Please translate using an informal tone (imagine you are talking with a friend about transformers 🤗).
* Please translate in a gender-neutral way.
* Add your translations to the folder called `<languageCode>` inside the [source folder](https://github.com/huggingface/transformers/tree/main/docs/source).
* Register your translation in `<languageCode>/_toctree.yml`; please follow the order of the [English version](https://github.com/huggingface/transformers/blob/main/docs/source/en/_toctree.yml).
* Once you're finished, open a pull request and tag this issue by including #issue-number in the description, where issue-number is the number of this issue. Please ping @stevhliu and @MKhalusova for review.
* 🙋 If you'd like others to help you with the translation, you can also post in the 🤗 [forums](https://discuss.huggingface.co/).

## Get Started section

- [ ] [index.md](https://github.com/huggingface/transformers/blob/main/docs/source/en/index.md) https://github.com/huggingface/transformers/pull/20180
- [ ] [quicktour.md](https://github.com/huggingface/transformers/blob/main/docs/source/en/quicktour.md) (waiting for initial PR to go through)
- [ ] [installation.md](https://github.com/huggingface/transformers/blob/main/docs/source/en/installation.md).

## Tutorial section
- [ ] [pipeline_tutorial.md](https://github.com/huggingface/transformers/blob/main/docs/source/en/pipeline_tutorial.md)
- [ ]  [autoclass_tutorial.md](https://github.com/huggingface/transformers/blob/master/docs/source/autoclass_tutorial.md)
- [ ]  [preprocessing.md](https://github.com/huggingface/transformers/blob/main/docs/source/en/preprocessing.md)
- [ ]  [training.md](https://github.com/huggingface/transformers/blob/main/docs/source/en/training.md)
- [ ]  [accelerate.md](https://github.com/huggingface/transformers/blob/main/docs/source/en/accelerate.md)
- [ ]  [model_sharing.md](https://github.com/huggingface/transformers/blob/main/docs/source/en/model_sharing.md)
- [ ]  [multilingual.md](https://github.com/huggingface/transformers/blob/main/docs/source/en/multilingual.md)

### PR DESCRIPTION
**Pull request by chenin-wang**
# What does this PR do?

Part of https://github.com/huggingface/transformers/issues/26803

## Before submitting
- [ ] This PR fixes a typo or improves the docs (you can dismiss the other checks if that's the case).
- [ ] Did you read the [contributor guideline](https://github.com/huggingface/transformers/blob/main/CONTRIBUTING.md#create-a-pull-request),
      Pull Request section?
- [ ] Was this discussed/approved via a Github issue or the [forum](https://discuss.huggingface.co/)? Please add a link
      to it if that's the case.
- [ ] Did you make sure to update the documentation with your changes? Here are the
      [documentation guidelines](https://github.com/huggingface/transformers/tree/main/docs), and
      [here are tips on formatting docstrings](https://github.com/huggingface/transformers/tree/main/docs#writing-source-documentation).
- [ ] Did you write any new necessary tests?

## Who can review?

Anyone in the community is free to review the PR once the tests have passed. Feel free to tag
members/contributors who may be interested in your PR.

@stevhliu and @MKhalusova

### CODE REVIEW

File: docs/source/zh/trainer.md [hunk 1]
```diff
@@ -0,0 +1,411 @@
...
+    eval_dataset=dataset["test"],
+    tokenizer=tokenizer,
+    data_collator=data_collator,
+    compute_metrics=compute_metrics,
+)
+
+trainer.train()
+```
+
+### checkpoints
```
Comment (stevhliu) [inline docs/source/zh/trainer.md:90]: Should this be translated?

```suggestion
### checkpoints
```

File: docs/source/zh/trainer.md [hunk 2]
```diff
@@ -0,0 +1,411 @@
...
+* [`~Trainer.get_test_dataloader`] 创建一个测试数据加载器
+* [`~Trainer.log`] 记录观察训练的各种对象的信息
+* [`~Trainer.create_optimizer_and_scheduler`] 如果没有在 `__init__`中定义，则创建一个优化器和学习率调度器；也可以分别指定为[`~Trainer.create_optimizer`]和[`~Trainer.create_scheduler`]
+* [`~Trainer.compute_loss`] 计算一批训练输入的损失
+* [`~Trainer.training_step`] 执行训练步骤
+* [`~Trainer.prediction_step`] 执行预测和测试步骤
+* [`~Trainer.evaluate`] 评估模型并返回评估指标
+* [`~Trainer.predict`] 在测试集上进行预测（如果提供了标签，则会输出指标）
+
+例如，如果您想自定义[' ~Trainer.compute_loss ']方法来使用加权损失。
```
Comment (stevhliu) [inline docs/source/zh/trainer.md:123]:
```suggestion
例如，如果您想自定义[`~Trainer.compute_loss`]方法来使用加权损失。
```

File: docs/source/zh/trainer.md [hunk 3]
```diff
@@ -0,0 +1,411 @@
...
+    eval_dataset=dataset["test"],
+    tokenizer=tokenizer,
+    data_collator=data_collator,
+    compute_metrics=compute_metrics,
+    callback=[EarlyStoppingCallback()],
+)
+```
+
+## 日志记录
+<Tip>
```
Comment (stevhliu) [inline docs/source/zh/trainer.md:180]:
```suggestion
## 日志记录

<Tip>
```

**Review by stevhliu — COMMENTED** [review]
Thanks for your contribution!

I can only review the technical structure and format of the docs because I can't read Chinese. Do you know of any other contributors who would be interested in reviewing your work? Also pinging @jiaqiw09 and @statelesshz (if you have any free time to review this 🙏 ) who have translated Chinese docs in the past.

File: docs/source/zh/trainer.md (hunk 1 above)
Comment (chenin-wang) [reply docs/source/zh/trainer.md:90]: In my opinion, translating checkpoints into Chinese is not as easy to understand as checkpoints.

Comment (chenin-wang) [reply docs/source/zh/trainer.md:90]: Just like the tokenizer

**Comment by stevhliu** [comment]
Hey @chenin-wang! Happy to merge once someone who can read Chinese has reviewed your translation! Do you know anyone who might be interested? The [forums](https://discuss.huggingface.co/) and [Discord](https://huggingface.co/join/discord) are also a good place to look for other collaborators :)

**Pull request closed without merging**